openrr-apps/src/bin/robot_teleop_keyboard.rs

```diff
@@ -0,0 +1,119 @@
+#![cfg_attr(windows, allow(dead_code, unused_imports))]
```

Comment:
I don't want to create this.
Is it difficult to modify robot_teleop.rs to support teleop-keyboard?
(That's the concept of arci.)

Reply:
I suppose I could add a property like `gamepad = "Gilrs" | "Keyboard"` to the teleop config.

```rust
pub enum GamepadKind {
    Gilrs,
    Keyboard,
}
```

...but I'm not sure that would be easy to use without #295.

Reply:
Even if it's not easy to use, I want you to do that.

And I noticed that Windows is not supported, right?

Reply:
> Even if it's not easy to use, I want you to do that.

Ok, will do.

> And I noticed that Windows is not supported, right?

Yeah, termios doesn't support windows.

Reply:
done